isos/base.sh

```diff
@@ -68,7 +68,7 @@ cp $DIR/base/*.repo $(rootfs_dir $PKGDIR)/etc/yum.repos.d/
 cp $DIR/base/yum.conf $(rootfs_dir $PKGDIR)/etc/yum/
 
 # install the core packages
-yum_cached -c $cache -u -p $PKGDIR install filesystem coreutils bash linux-esx --nogpgcheck -y
+yum_cached -c $cache -u -p $PKGDIR install filesystem coreutils bash linux-esx photon-release --nogpgcheck -y
```

Comment:
This should be added into the appliance-staging.sh, not the base. The entire initramfs is blown away after tether launches in the bootstrap.